at_commons/lib/src/keystore/at_key.dart

```diff
@@ -269,25 +276,79 @@ class PrivateKey extends AtKey {
 }
 
 class Metadata {
+  /// Represents the time in milliseconds beyond which the key expires
   int? ttl;
+
+  /// Represents the time in milliseconds from when the key becomes active
   int? ttb;
+
+  /// Represents the time frequency in seconds when the cached key gets refreshed
   int? ttr;
+
+  /// CCD (Cascade Delete) means if a shared key is deleted, then the corresponding cached key will also be deleted
+  ///
+  /// When set to true, on deleting the original key, the corresponding cached key will be deleted,
+  /// When set to false, the cached key remains even after the original key is deleted.
```

Comment:
Are there any tests which verify this functionality? i.e. is this metadata honoured?